bazel/defs.bzl

```diff
@@ -89,10 +89,25 @@ def proxy_wasm_cc_binary(
             "@proxy_wasm_cpp_sdk//:proxy_wasm_intrinsics_js",
         ],
         linkopts = linkopts + [
+            # Setting to indicate module is a "reactor library" without a main() entry point:
+            # https://emscripten.org/docs/tools_reference/settings_reference.html#standalone-wasm
             "--no-entry",
+            # File listing additional functions that Emscripten should expect to be implemented by the host:
+            # https://emscripten.org/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.html#implement-c-in-javascript
             "--js-library=$(location @proxy_wasm_cpp_sdk//:proxy_wasm_intrinsics_js)",
+            # Emit Wasm module that can run without JavaScript
             "-sSTANDALONE_WASM",
+            # Give host code access to Emscripten's _malloc() function
             "-sEXPORTED_FUNCTIONS=_malloc",
+            # Allow allocating memory past initial heap size
+            "-sALLOW_MEMORY_GROWTH=1",
+            # Total stack size (fixed). Emscripten default stack size changed from 5MiB to 64KiB in
+            # 3.1.27: https://github.com/emscripten-core/emscripten/blob/main/ChangeLog.md,
+            # https://github.com/emscripten-core/emscripten/pull/18191. We pick 1MiB somewhat
+            # arbitrarily, since it is gives a little more room and is easy to remember.
+            "-sSTACK_SIZE=1MB",
```

Comment:
Feels kind of wasteful for a function/plugin, but that's just a gut feeling.
Real world system defaults have a wide range: https://stackoverflow.com/a/1826072

Do we have real world evidence that Emscripten's 64 KiB default is insufficient?

Reply:
@PiotrSikora We can put away the worries about protobuf (and CEL) in Wasm away from Istio perspective. Istio is not using Wasm for production and the Google filters are perma-null Wasm. I think smaller overhead is better to match Lua, so 1MB or even less makes sense to me. One thing to worry is that standard library functions hide some deep recursion that can manifest on the data plane in a surprising way. I don't know how to protect against that in a general purpose language.